spec/controllers/events_controller_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe EventsController, type: :controller do
+  describe 'GET #show' do
+    it 'returns http success' do
+      get :show
+      expect(response).to have_http_status(:success)
```

Comment:
The test is failing, why? This is a hint to look deeper in your code.

Let's do a manual test here:
* Make sure you're logged out
* Navigate to an event show page, i.e. `http://localhost:3000/events/1`

What happened? Current user doesn't exist? Of course it doesn't. You're not logged in! That's why this test is failing.

Furthermore,  I'd suggest looking at your routes with `$ rails routes | grep user` and checking to see if everything makes sense in there. There's an extra route there for `show` as `/users/show/` but by convention it should be `/users/show/:id`.

[`REQUIRED`]